packages/start-slicemachine/bin/start-slicemachine.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+import("../dist/cli.cjs");
```

Comment:
Are imports supported in node v14?

Reply:
Yes, dynamic imports (`import()`) are supported as of Node.js 13.2.0. In Node.js, they are the only way to import ES modules from CJS code.